vm/vm/src/Tevm.ts

```diff
@@ -40,9 +41,14 @@ export type Tevm = import('@tevm/api').Tevm & {
 	 * Internal instance of the EVM. Can be used for lower level operations
 	 * but is not guaranteed to stay stable between versions
 	 */
-	readonly _evm: import('@ethereumjs/evm').EVM
+
+	readonly _evm: TevmEvm
 	/**
 	 * Creates a HTTP handler for the tevm vm
 	 */
 	readonly createHttpHandler: () => ReturnType<typeof createHttpHandler>
 }
+
+export class TevmEvm extends EVM {
+	public declare stateManager: TevmStateManager
+}
```

Comment:
If you replace TevmStateManger with a util function you can remove this

Reply:
Keeping this class until we can implement this function in ethereumJs